gazebo/msgs/msgs.cc

```diff
@@ -1817,6 +1817,11 @@ namespace gazebo
         result.mutable_ray()->CopyFrom(msgs::RaySensorFromSDF(
             _sdf->GetElement("ray")));
       }
+      else if (type == "gpu_ray")
+      {
+        result.mutable_ray()->CopyFrom(msgs::RaySensorFromSDF(
+            _sdf->GetElement("gpu_ray")));
```

Comment:
ok, I think I know the problem; try changing `"gpu_ray"` to `"ray"`; they both use the same SDFormat elements, just a different attribute

Reply:
It's weird. Both have the same structure and `RaySensorFromSDF` expects an `ElementPtr` (nothing specific). I made the change and I'll take a look to CI results.

Reply:
I think the problem is that a `gpu_ray` element is not define in the SDFormat specification, only `ray`:

* https://github.com/osrf/sdformat/blob/sdformat11_11.0.0/sdf/1.7/sensor.sdf#L65

thanks for making the change!